instagram/src/components/CommentSection/CommentSection.js

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import './CommentSection.css'
+import SingleComment from './SingleComment'
+
+
+const CommentSection = (props) =>{
+    return(
+        <div>
```

Comment:
Another option instead of using the div, is using fragments `<>` to contain child elements in a React component